sqlglot/jsonpath.py

```diff
@@ -146,6 +146,21 @@ def _parse_bracket() -> exp.JSONPathPart:
 
         return node
 
+    def _parse_var_text() -> str:
```

Comment:
May be worth putting a quick comment to explain this briefly.

Reply:
Good idea, let me know if that will suffice.